dataland-frontend/tests/e2e/specs/sfdr/DataUpload.ts

```diff
@@ -0,0 +1,106 @@
+import { describeIf } from "@e2e/support/TestUtility";
+import { admin_name, admin_pw, getBaseUrl } from "@e2e/utils/Cypress";
+import { DataTypeEnum, SfdrData } from "@clients/backend";
+import { getKeycloakToken } from "@e2e/utils/Auth";
+import { generateDummyCompanyInformation, uploadCompanyViaApi } from "@e2e/utils/CompanyUpload";
+import { addsReportsToSfdrDataViaForm, uploadOneSfdrDataset } from "@e2e/utils/SfdrUpload";
+import { FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { submitButton } from "@sharedUtils/components/SubmitButton";
+import { toggleRowGroup } from "@sharedUtils/components/ToggleRowFunction";
+
+let companiesWithSfdrData: Array<FixtureData<SfdrData>>;
+let testSfdrCompany: FixtureData<SfdrData>;
+before(function () {
+  cy.fixture("CompanyInformationWithSfdrPreparedFixtures").then(function (jsonContent) {
+    companiesWithSfdrData = jsonContent as Array<FixtureData<SfdrData>>;
+    testSfdrCompany = getPreparedFixture("CompanyInformationWithSfdrData", companiesWithSfdrData);
+  });
+});
+describeIf(
+  "As a user, I expect that the upload form works correctly when editing and uploading a new SFDR dataset",
+  {
+    executionEnvironments: ["developmentLocal", "ci", "developmentCd"],
+  },
+  function () {
+    beforeEach(() => {
+      cy.ensureLoggedIn(admin_name, admin_pw);
+    });
+
+    //TODO Rename file to DataIntegrity after everything is finished
```

Comment:
Why? Renaming a newly created file has never caused problems with git in the past of the project as far as I remember.

Reply:
Wanted to make sure that no one else is working on this at the same time, because i am not sure if then merge conflicts could arise.

Reply:
Renamed

Reply:
TODO is still in there

Reply:
removed

Reply:
👍